doc/u-boot-env.md

```diff
@@ -32,10 +32,15 @@ setenv ethaddr 12:34:56:78:9A:BC
 
 ## TFTP boot
 ```
+# ./build-domd/tmp/deploy/images/YOUR_BOARD/xen-uImage
```

Comment:
It would be nice to have this information, but:

1. Not in this commit. This commit is devoted to NFS boot options
2. In another place, because this section is describes network boot, but this files (like Xen image) are used in all types of boot, so it better to describe their location in a separate section.

Reply:
done